main/index.js

```diff
@@ -13,11 +24,11 @@ process.env['BUNDLE_LOCATION'] = process.env.BUNDLE_LOCATION || path.resolve(__d
 // app.commandLine.appendSwitch('enable-transparent-visuals', true)
 // if (process.platform === 'linux') app.commandLine.appendSwitch('disable-gpu', true)
 
-const log = require('electron-log')
+const Sentry = require('@sentry/electron')
```

Comment:
what's this line needed for?

Reply:
See line 88.  I originally had the sentry uncaught exception changes in this PR and then split them out